src/controllers/hashtagsController.js

```diff
@@ -0,0 +1,12 @@
+import { tagsRepository } from '../repositories/index.js';
+
+export const getAllTagsMentioned = async (req, res) => {
+  try {
+    const tags = await tagsRepository.getTags();
+    return res.status(200).send(tags);
```

Comment:
When you simply send something there is no need to add status 200 because its default. Also, when you are sending a json object its better to use `res.json(tags)`.

Reply:
Done.